tests/config.py

```diff
@@ -16,6 +16,9 @@
 import os
 import getpass
 
+SSH_USE_EXISTING_AGENT = bool(distutils.util.strtobool(
```

Comment:
Could you please add a comment about the intended use

Reply:
Done

Reply:
I think we need to point out that the intention is to make keys available to libvirt. Otherwise this could imply that an existing key will be loaded onto the host. However, what really happens is a key is still generated and then added to the agent.

It should also be noted that the added key isn't removed from the agent. Perhaps this is something we should clean up?